activejob/lib/active_job/queue_adapters.rb

```diff
@@ -96,9 +99,36 @@ module ActiveJob
   #
   # N/A: The adapter does not run in a separate process, and therefore doesn't
   # support retries.
+  #
+  # === Async and Inline Queue Adapters
+  #
+  # Active Job has two built-in queue adapters intended for development and
+  # testing: +:async+ and +:inline+.
+  #
+  # ==== Async Job Queue Adapter
+  #
+  # When enqueueing jobs with Async Job each job will be executed
+  # asynchronously on a +concurrent-ruby+ thread pool. All job data
+  # is retained in memory. Because job data is not saved to a persistent
+  # datastore there is no additional infrastructure needed and most jobs
+  # will process very quickly. The lack of persistence, however, means
+  # that all unprocessed jobs will be lost on application restart. This
+  # makes in-memory queue adapters unsuitable for most production
+  # environments but excellent for development and testing. See the
+  # {api documentation}[http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/AsyncAdapter.html]
+  # for more information.
+  #
+  # ==== Inline Queue Adapter
+  #
+  # When enqueueing jobs with the Inline adapter the job will be executed
+  # immediately on the calling thread. Job scheduling is not supported. This
+  # adapter is intended for use in tests. For more information see the
+  # {api documentation}[http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters/AsyncAdapter.html]
```

Comment:
This doesn't seem to be the right link and "for more information" is redundant here.

Actually, I guess you can simply remove this part and the previous one as well (i.e. "Async Job Queue Adapter"). They don't bring anything and may get out of sync when these adapters get updated. What do you think ?